test/langtools/tools/javac/versions/Versions.java

```diff
@@ -89,7 +89,8 @@ static enum SourceTarget {
         EIGHTEEN(false,  "62.0", "18", Versions::checksrc18),
         NINETEEN(false,  "63.0", "19", Versions::checksrc19),
         TWENTY(false,    "64.0", "20", Versions::checksrc20),
-        TWENTY_ONE(false,"65.0", "21", Versions::checksrc20);
+        TWENTY_ONE(false,"65.0", "21", Versions::checksrc20),
+        TWENTY_TWO(false,"66.0", "22", Versions::checksrc20);
```

Comment:
when do these get updated?

Reply:
Ah, when new non-preview language structures are added. With two new language features in JDK 21, looks like it is time for me to add another one; thanks.

Reply:
Better to fix this omission in JDK 21; please see https://github.com/openjdk/jdk/pull/14229.

Reply:
> Better to fix this omission in JDK 21; please see #14229.

Addressed by merge of JDK-8309134 into JDK 22; thanks.